conf/testOnlyDoNotUseInAppConf.routes

```diff
@@ -10,10 +10,13 @@
 # Failing to follow this rule may result in test routes deployed in production.
 
 + nocsrf
-POST        /manage-transit-movements/test-only/manage-transit-movements/to-core-message        controllers.testOnly.TestOnlyRouterController.toCoreMessage()
+POST        /manage-transit-movements/test-only/arrival-notification-to-core     controllers.testOnly.TestOnlyRouterController.arrivalNotificationMessageToCore()
 
 + nocsrf
-POST        /manage-transit-movements/test-only/manage-transit-movements/from-core-message      controllers.testOnly.TestOnlyRouterController.fromCoreMessage()
+POST        /manage-transit-movements/test-only/unloading-remarks-to-core        controllers.testOnly.TestOnlyRouterController.unloadingRemarksMessageToCore()
```

Comment:
This endpoint should accept a generic message so it shouldn't be limited to the context of submitting an Unloading Remark